### ISSUE TITLE
Command `docker compose run --rm auto-gpt serve` does not work

### ISSUE THREAD

**Issue by amittos** [Significant-Gravitas/AutoGPT#6555]
### ⚠️ Search for existing issues first ⚠️

- [X] I have searched the existing issues, and there is no existing issue for my problem

### Which Operating System are you using?

Docker

### Which version of AutoGPT are you using?

Latest Release

### Do you use OpenAI GPT-3 or GPT-4?

GPT-4

### Which area covers your issue best?

Installation and setup

### Describe your issue.

I'm following the instructions of how to run AutoGPT via Docker from [here](https://docs.agpt.co/autogpt/usage/). 

Running `docker compose run --rm auto-gpt` works exactly as expected. I.e., AutoGPT runs via the terminal and behaves as expected. 

However, running `docker compose run --rm auto-gpt serve` does not work. Running this command gives me the following error: 

```
Usage: python -m autogpt [OPTIONS] COMMAND [ARGS]...
Try 'python -m autogpt --help' for help.

Error: No such command 'serve'.
```

According to [this](https://docs.agpt.co/autogpt/usage/#command-line-interface) page, this should not happen.

**Comment by Bentlybro** [comment]
The docker build has recently been updated to fix this https://hub.docker.com/r/significantgravitas/auto-gpt/tags (docker pull significantgravitas/auto-gpt:v0.5.0) if this works please let me know and we can close this issue, if not still let me know and we can investigate more.

**Comment by Pwuts** [comment]
`serve` did not exist yet in `v0.4.7`. Yesterday we released `v0.5.0`, in which it does work:
![image](https://github.com/Significant-Gravitas/AutoGPT/assets/12185583/5b9e7d10-a978-49dc-bd51-a48bc6481f7c)

Let us know if you have any more issues!

**Comment by gpechieu** [comment]
I have tried with the guide to use with Docker and always get the same result, it's not the exposed ports. And when they are manually exposed, the following error occurs:

{"detail":"Not Found"}

Could it be something related to the warning that appears "Frontend not found. /frontend/build/web does not exist. The frontend will not be available."
Just like the previous response, it also appears to me.
